src/libraries/System.IO.IsolatedStorage/tests/AssemblyInfo.cs

```diff
@@ -9,3 +9,4 @@
 // create unique identities for every test to allow every test to have
 // it's own store.
 [assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly, DisableTestParallelization = true)]
+[assembly: ActiveIssue("https://github.com/dotnet/runtime/issues/37669", TestPlatforms.Browser)]
```

Comment:
Do we expect to fix IsolatedStorage for Wasm, or do we expect it to be unsupported permanently?

Linking the disabled tests to active issues suggests that we plan fixing it, but the rest of the code delta suggests that the plan is to not support it.

Reply:
yeah this should be using `[assembly: SkipOnMono("System.IO.IsolatedStorage is not supported on WASM", TestPlatforms.Browser)]` since we don't plan to enable it.

Reply:
I put this as an `[ActiveIssue]` since crypto plays a part in this library not working right.  I'd expect we revisit once there's an acceptable solution there.

Reply:
Hm I thought we discussed last night and concluded that this library makes little sense on WASM even though we could likely make it work with some effort. I think we shouldn't tie this to the crypto ActiveIssue if it might never happen.

Reply:
I changed it to `SkipOnMono`.  When i first looked at it, I felt that `[ActiveIssue]` could be a reminder to reconsider.  But, honestly, that could happen at any time ;-).